sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -660,6 +665,8 @@ int gnrc_netif_ipv6_addr_add_internal(gnrc_netif_t *netif,
             gnrc_ipv6_nib_pl_set(netif->pid, addr, pfx_len,
                                  UINT32_MAX, UINT32_MAX);
         }
+
+        _nib_bus_post(netif, GNRC_IPV6_NIB_EVENT_ADDR_VALID, &netif->ipv6.addrs[idx]);
```

Comment:
```
/home/mlenders/Repositories/RIOT-OS/RIOT/sys/net/gnrc/netif/gnrc_netif.c: In function ‘gnrc_netif_ipv6_addr_add_internal’:
/home/mlenders/Repositories/RIOT-OS/RIOT/sys/net/gnrc/netif/gnrc_netif.c:669:9: error: implicit declaration of function ‘_nib_bus_post’ [-Werror=implicit-function-declaration]
  669 |         _nib_bus_post(netif, GNRC_IPV6_NIB_EVENT_ADDR_VALID, &netif->ipv6.addrs[idx]);
      |         ^~~~~~~~~~~~~
```

Reply:
arg I would have almost gotten away with keeping this inside an internal header if I had restricted it's use to stuff inside `nib/`.

So should I create a new internal header for this or move it to a global one?

Reply:
Move it to `net/gnrc/netif/internal.h` I would say.

Reply:
Ok, moved it there.